src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -287,10 +287,17 @@
       <_BinPlaceItemName Condition="'$(_BinPlaceItemName)' == ''">BinPlaceItem</_BinPlaceItemName>
     </PropertyGroup>
 
+    <ItemGroup>
+      <_BinPlaceItems Include="@(BinPlaceDir)">
+        <TargetPath Condition="'%(BinPlaceItem.TargetPath)' == ''">%(BinPlaceItem.Filename)%(BinPlaceItem.Extension)</TargetPath>
```

Comment:
I can't get the attribute evaluation working with a dynamic item group name. I tried something like `'%($(_BinPlaceItemName).TargetPath)' == ''` and `'%($(_BinPlaceItemName.TargetPath))' == ''` and various other combinations. I also can't find any useful resources online...